test/z85_vectors.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Test vector for Z85 (from http://rfc.zeromq.org/spec:32/Z85/)
+ * | 0x86 | 0x4F | 0xD2 | 0x6F | 0xB5 | 0x59 | 0xF7 | 0x5B | encodes as
+ * | H | e | l | l | o | W | o | r | l | d |
+ */
+sjcl.test.vector.z85 =
+[
+[[0x864FD26F, 0xB559F75B], "HelloWorld"]
```

Comment:
Please use `sjcl.codec.hex.toBits("0x864FD26FB559F75B")` so we have a full test here (you can also use the string here and transform it to bits in the test itself. This way even if the bitarray implementation changes, this test would still work (or better fail if necessary).